rules/crowdstrike_rules/crowdstrike_real_time_response_session.py

```diff
@@ -1,29 +1,32 @@
-from panther_base_helpers import deep_get
+from panther_base_helpers import get_crowdstrike_field
+
+MATCHES = []
 
 
 def rule(event):
     return (
-        deep_get(event, "unknown_payload", "ExternalApiType", default="<unknown-ExternalApiType>")
+        get_crowdstrike_field(event, "ExternalApiType", default="<unknown-ExternalApiType>")
```

Comment:
in the case of the old log type, we're going to end up calling `deep_get` three times on every event that comes into this rule. have we profiled the impact of this?

Reply:
Seems like a dictionary "get" operation in python (provided that we already have set up the objects of course) is in the order of [microseconds](https://docs.python.org/3/library/timeit.html), so I wouldn't be worried about this. Also this will work for the old-log-type only for a limited time.

Reply:
With the help of @georgepsarakis I got this:
![image](https://user-images.githubusercontent.com/777619/215094635-a40857aa-d53f-4219-a778-30704a66a400.png)

Reply:
thank you!